examples/tv-app/android/java/ContentAppAttributeDelegate.cpp

```diff
@@ -0,0 +1,64 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ * @brief Contains Implementation of the ContentAppAttributeDelegate
+ */
+
+#include "ContentAppAttributeDelegate.h"
+#include <app-common/zap-generated/cluster-objects.h>
+#include <app/AttributeAccessInterface.h>
+#include <jni.h>
+#include <lib/support/CHIPJNIError.h>
+#include <lib/support/JniReferences.h>
+#include <lib/support/JniTypeWrappers.h>
+#include <zap-generated/endpoint_config.h>
+
+namespace chip {
+namespace AppPlatform {
+
+using LaunchResponseType = chip::app::Clusters::ContentLauncher::Commands::LaunchResponse::Type;
+
+const char * ContentAppAttributeDelegate::Read(const chip::app::ConcreteReadAttributePath & aPath)
+{
+    if (aPath.mEndpointId >= FIXED_ENDPOINT_COUNT)
```

Comment:
if you switch the order of conditions, indent is reduced and code is easier to read. Do this:

```
if (aPath.mEndpointId < FIXED_ENDPOINT_COUNT) {
    return "";
}

JNIEnv * env = JniReferences::GetInstance().GetEnvForCurrentThread();
// ....
```

Reply:
Updated.